File: Back/src/ProEventos.Domain/Event.cs
using System;
using System.Collections.Generic;

namespace ProEventos.Domain
{
    public class Event
    {
        public int Id { get; set; }
        public string Location { get; set; }
        public DateTime? EventDate { get; set; }
        public string Theme { get; set; }
        public int NumberOfPeople { get; set; }
        public string ImageURL { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public IEnumerable<Batch> Batches { get; set; }
        public IEnumerable<SocialMedia> SocialMedias { get; set; }
        
        public IEnumerable<SpeakerEvent> SpeakerEvents { get; set; }
    }
}

File: Front/ProEventos-App/src/app/models/Event.ts
import { Batch } from './Batch';
import { SocialMedia } from './SocialMedia';
import { Speaker } from './Speaker';

export interface Event{
  Id: number;
  Location: string;
  EventDate: Date;
  Theme: string;
  NumberOfPeople: number;
  ImageURL: string;
  Phone: string;
  Email: string;
  Batches: Batch[];
  SocialMedias: SocialMedia[];
  SpeakerEvents: Speaker[];
    }
